research/picai/README.md

```diff
@@ -94,35 +94,82 @@ The raw annotations differ in spatial resolution across annotators. Specifically
+In addition to to Raw Dataset, a Preprocessed Dataset has been provided on the cluster that was generated using scripts described in the following section. The preprocessed dataset has been prepared in the format the [nnUNet framework](https://github.com/MIC-DKFZ/nnUNet) making it easy to apply out of the box to the dataset. For more information about nnUNet as it relates to our experimentation on the PICAI dataset, refer to [nnUNet_overview.md](../picai/nnunet_scripts/nnunet_overview.md). Additional preprocessing utilities and training baselines are available in the [picai_prep package](https://github.com/DIAGNijmegen/picai_prep) and [picai_baseline package](https://github.com/DIAGNijmegen/picai_baseline). These packages were provided by the PICAI competition to give a starting point for practitioners to preprocess the data from the Public Training and Development Dataset and train a simple model for the csPCa task.
 
 ### Preprocessing
-The Preprocessed Dataset was generated by following the data preparation pipeline of nnU-Net with the following exception:
+The Preprocessed Dataset was generated by running the `prepare_annotations.py` and `prepare_data.py` scripts sequentially. These scripts assume that the dataset is formatted identically to the raw dataset described above.
+
+`prepare_annotations.py` is a simple script that copies the human and ai-derived annotations into a specified folder. An example invocation is as follows:
+
+```
+python research/picai/preprare_annoations.py --human_annotations_dir /path/to/human/annotations --ai_annotations_dir /path/to/ai/annotations --annotations_write_dir /path/to/write/dir
+```
+
+`prepare_data.py` is the main preprocessing script that takes in a number of arguments related to the location of the raw dataset and details about the preprocessing and produces a preprocessed dataset with an associated dataset overview file. Here is an example invocation:
+```
+python research/picai/prepare_data.py --scans_read_dir /path --annotations_read_dir /path --scans_write_dir /path --overview_write_dir /path --size 20 256 256  --spacing 3 0.5 0.5 --num_threads 4 --splits_path /path
+```
+The arguments are defined accordingly:
+- **scans_read_dir**: Base directory containing a number of subdirectories - each with one or more scans.
+- **annotations_read_dir**: Base directory containing a number of annotations files.
+- **scans_write_dir**: Base directory to write all preprocessed scans.
+- **annotations_write_dir**: Base directory to write all preprocessed annotations.
+- **overview_write_dir**: Base directory to write all dataset overviews (more details below).
+- **size**: Desired dimensions of preprocessed images.
```

Comment:
Perhaps a quick comment on the format and what they mean. Triplet of the form Depth Weight Height in pixels.